syndeo-template/src/main/java/com/google/cloud/syndeo/transforms/sql/SyndeoScalarSqlTransformSchemaTransformProvider.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.syndeo.transforms.sql;
+
+import com.google.auto.service.AutoService;
+import com.google.auto.value.AutoValue;
+import java.util.Collections;
+import java.util.List;
+import org.apache.beam.sdk.extensions.sql.expansion.SqlTransformSchemaTransformProvider;
```

Comment:
Would it be better to wrap `SyndeoSqlSchemaTransformProvider`. It seems like this would just generate the SQL query to pass to that, letting it fill in fields like the dialect.

Reply:
done